sdk/src/feature_set.rs

```diff
@@ -785,6 +785,14 @@ pub mod deprecate_unused_legacy_vote_plumbing {
     solana_sdk::declare_id!("6Uf8S75PVh91MYgPQSHnjRAPQq6an5BDv9vomrCwDqLe");
 }
 
+pub mod dont_load_executable_accounts {
```

Comment:
please don't introduce two feature gates in the same pr. this is indicative of a change set that should be split up (as is the line change total)